tests/utils/configmaps.go

```diff
@@ -1,50 +1,66 @@
 package utils
 
 import (
+	"strings"
+
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 
 	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/util"
 )
 
 const registryHostname = RegistryHostName + "." + RegistryHostNs
 
-// CopyRegistryCertConfigMap copies the test registry ConfigMap
+// CopyRegistryCertConfigMap copies the test registry configmap
 func CopyRegistryCertConfigMap(client kubernetes.Interface, destNamespace string) (string, error) {
-	err := CopyConfigMap(client, RegistryHostNs, RegistryConfigMap, destNamespace, RegistryConfigMap)
+	n, err := CopyConfigMap(client, RegistryHostNs, RegistryCertConfigMap, destNamespace, "")
 	if err != nil {
 		return "", err
 	}
-	return RegistryConfigMap, nil
+	return n, nil
+}
+
+// CopyFileHostCertConfigMap copies the test file host configmap
+func CopyFileHostCertConfigMap(client kubernetes.Interface, destNamespace string) (string, error) {
+	n, err := CopyConfigMap(client, FileHostNs, FileHostCertConfigMap, destNamespace, "")
+	if err != nil {
+		return "", err
+	}
+	return n, nil
 }
 
 // CopyConfigMap copies a ConfigMap
-func CopyConfigMap(client kubernetes.Interface, srcNamespace, srcName, destNamespace, destName string) error {
+func CopyConfigMap(client kubernetes.Interface, srcNamespace, srcName, destNamespace, destName string) (string, error) {
 	src, err := client.CoreV1().ConfigMaps(srcNamespace).Get(srcName, metav1.GetOptions{})
 	if err != nil {
-		return err
+		return "", err
+	}
+
+	if destName == "" {
+		destName = srcName + "-" + strings.ToLower(util.RandAlphaNum(8))
```

Comment:
Maybe I am missing something, why are we creating a config map with a partially random name? Its getting copied to a new namespace which already has a randon name, so what is the reason to have the same thing for the config map?

Reply:
Not strictly necessary given that tests are running serially.  But I like the idea of each test having its own distinct data.  Which will be necessary if we ever run tests in parallel (which would be nice)

Reply:
But each test creates its own namespace, which will be unique, even the ones in a 'table'

Reply:
Okay, I was confused.  I thought that a single namespace was created for all of the transport tests.  Did not see that the framework adds a before/after each calback that creates/deletes the namespace.

I still think this randomization is a nice to have.

Reply:
It doesn't bother me that much, just wanted to point it out.